### PR TITLE
Count Set Bits

### PR DESCRIPTION
**Pull request by achalesh27022003**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

(Write your answer here.)

### Description

(Write your answer here.)

### Checklist

- [ ] I've read the contribution guidelines.
- [ ] I've checked the issue list before deciding what to submit.
- [ ] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

#1783

### CODE REVIEW

File: C-Plus-Plus/other/BrianKenighan'sAlgorithm.cpp [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+        return count;
+}
+int main()
+{
+    int i;
+    cout << "Enter a number" << endl;
+    cin >> i;
+    cout <<"The number of set bits or 1s is : " << countSetBits(i);
+    return 0;
+}
```
Comment (HarshCasper) [inline C-Plus-Plus/other/BrianKenighan'sAlgorithm.cpp:30]: Code plagiarized from [here](https://www.geeksforgeeks.org/count-set-bits-in-an-integer/)

File: C-Plus-Plus/other/CountSetBits.cpp [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+}
+ 
+int main()
+{
+    int n;
+    cout << "Enter a number" << endl;
+    cin >> n;
+    cout <<"The number of set bits or 1s is: " countSetBits(n);
+    return 0;
+}
```
Comment (HarshCasper) [inline C-Plus-Plus/other/CountSetBits.cpp]: Code plagiarized from [here](https://www.geeksforgeeks.org/count-set-bits-in-an-integer/)

**Comment by achalesh27022003** [comment]
Please tell how I can remove plagriarism from an algorithm? I took help from there & learnt so applied it. Please guide next what to do sir.

**Comment by HarshCasper** [comment]
> Please tell how I can remove plagriarism from an algorithm? I took help from there & learnt so applied it. Please guide next what to do sir.

Learning from a source does not give a right to plagiarize that source. I'm closing the PR and would beseech you to come up with your own code even though you can learn from any resources. Pushing plagiarized code is count as malicious to the overall health of the project.

**Pull request closed without merging**